src/test/java/calculator/Calculator.java

```diff
@@ -0,0 +1,51 @@
+package calculator;
+
+import java.util.ArrayList;
+
+public class Calculator {
+
+    private static final String DEFAULT_DELIM = " ";
+
+    public long eval(String s) {
+        return eval(s, DEFAULT_DELIM);
+    }
+
+    public long eval(String s, String delim) {
+        if (s == null) {
+            throw new IllegalArgumentException("null");
+        }
+
+        String[] tokens = s.split(delim);
+
+        TokenSequence sequence = new TokenSequence();
+        ArithmeticOperator operator = null;
+        ArrayList<Long> operands = new ArrayList<>();
```

Comment:
```suggestion
        List<Long> operands = new ArrayList<>();
```
마찬가지로 List 로 선언하는건 어떨까요 😄